docs/providers.rst

```diff
@@ -235,6 +235,39 @@ The provider is OAuth2 based. More info:
 
 Note: This is not the same as the Mozilla Persona provider below.
 
+The following Firefox Accounts settings are available::
+
+    SOCIALACCOUNT_PROVIDERS = \
+        {'fxa':
+           {'DEPLOYMENT': 'production',
+            'ON_WHITELIST': True,
+            'SCOPE': ['profile'],
+            'OAUTH_ENDPOINT': 'https://oauth.accounts.firefox.com/v1',
+            'PROFILE_ENDPOINT': 'https://profile.accounts.firefox.com/v1'}}
+
+DEPLOYMENT:
+    The
+    `Firefox Accounts deployment <https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Firefox_Accounts/Introduction#Firefox_Accounts_deployments>`_
+    to authenticate against. Possible values are "production" (default),
+    "stage", "stable", and "latest".
+
+ON_WHITELIST:
+    True (default) if your application is on the Mozilla trusted whitelist,
+    False if it is not, and has limited access to profile data.
+
+SCOPE:
+    Requested OAuth2 scope. Default is ['profile'] if ON_WHITELIST, and
+    ['profile:email', 'profile:uid'] if not.
```

Comment:
``SCOPE`` is part of the [base ``oauth2``](https://github.com/pennersr/django-allauth/blob/master/allauth/socialaccount/providers/oauth2/provider.py#L28-L36) provider. If the scope is provided, then it is used, otherwise the return from ``get_default_scope()`` is used. I'm following the example of the [facebook provider docs](https://github.com/pennersr/django-allauth/blob/master/docs/providers.rst#facebook), which mention ``SCOPE`` even without special handling in the provider.